es/10_1_Understanding_the_Foundation_of_P2SH.md

```diff
@@ -0,0 +1,99 @@
+# 10.1: Comprender la Base de P2SH
+
+Sabe que los scripts de Bitcoin se pueden usar para controlar el canje de UTXOs. El siguiente paso es crear sus propios scripts ... pero eso requiere una técnica muy específica.
+
+## Conozca los Estándares de Bitcoin
+
+Aquí está la trampa para usar los scripts de Bitcoin: por razones de seguridad, la mayoría de los nodos de Bitcoin solo aceptarán seis tipos de transacciones de Bitcoin "estándar".
+
+* __Pagar a Clave Pública (Pay to Public Key, P2PK)__ — Una transacción antigua y obsoleta (`<pubKey> OP_CHECKSIG`) que ha sido reemplazada por la mejor seguridad de P2PKH.
+* __Pagar al Hash de Clave Pública (Pay to Public Key Hash, P2PKH)__ — Una transacción (`OP_DUP OP_HASH160 <pubKeyHash> OP_EQUALVERIFY OP_CHECKSIG`) que paga el hash de una clave pública.
+* __Pagar Para Ser Testigo de Hash de Clave Pública (Pay to Witness Public Key Hash, P2WPKH)__ — El tipo más nuevo de transacción de clave pública. Es solo (`OP_0 <pubKeyHash`) porque depende del consenso del minero para funcionar, como se dscribe en [§9.5](09_5_Scripting_a_P2WPKH.md).
```

Comment:
_Pagar al testigo del Hash de la Clave Pública (

Reply:
Implemented correction as suggested for line 10. Do lines 9 or 11 need to be similarly changed as well? a la Clave; Para Ser Testigo del Hash de la Clave Pública. Or are they fine as they are?